python/cli/prelude_cli/views/scm.py

```diff
@@ -810,15 +810,27 @@ def delete_report(controller, report_id):
 
 @report.command("put")
 @click.option(
-    "--report_data", required=True, type=str, help="Report data in JSON format"
+    "--report_data",
+    required=True,
```

Comment:
don't make this required anymore - just check that either report_data or _report_file is set in the code